bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/PageChangeListener.java

```diff
@@ -231,18 +231,20 @@ public void changeStateTo(Item item, State state) {
                 event.valuecolor = itemUIRegistry.getValueColor(w);
                 event.widgetId = itemUIRegistry.getWidgetId(w);
                 event.visibility = itemUIRegistry.getVisiblity(w);
-                // event.item contains data from the item including its state (in event.item.state)
-                String widgetTypeName = w.eClass().getInstanceTypeName()
-                        .substring(w.eClass().getInstanceTypeName().lastIndexOf(".") + 1);
-                boolean drillDown = "mapview".equalsIgnoreCase(widgetTypeName);
-                Predicate<Item> itemFilter = (i -> i.getType().equals(CoreItemFactory.LOCATION));
-                event.item = EnrichedItemDTOMapper.map(item, drillDown, itemFilter, null, null);
+                if (!skipWidget) {
+                    // event.item contains data from the item including its state (in event.item.state)
+                    String widgetTypeName = w.eClass().getInstanceTypeName()
+                            .substring(w.eClass().getInstanceTypeName().lastIndexOf(".") + 1);
+                    boolean drillDown = "mapview".equalsIgnoreCase(widgetTypeName);
+                    Predicate<Item> itemFilter = (i -> i.getType().equals(CoreItemFactory.LOCATION));
+                    event.item = EnrichedItemDTOMapper.map(item, drillDown, itemFilter, null, null);
 
-                // event.state is an adjustment of the item state to the widget type.
-                event.state = itemUIRegistry.convertState(w, item, state).toFullString();
```

Comment:
... but still keep this null if `skipWidget` is true. That way, (AFAICT) the correct item will be sent, but no explicit state will be sent with the event.